### DISCUSSION TITLE
How tag editor looks? I do not see it in 0.9.5

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [nulloy/nulloy#202] [Q&A]
.

**Comment by noscript** [comment]
Right click on a track in playlist -> **Tag Editor**

![image](https://user-images.githubusercontent.com/717109/179395163-df6c642c-31d7-447f-938b-11856dd86f11.png)

**Reply by vitaly-zdanevich** [reply]
In the right click menu on the track I see only 

```
Reveal in File Manager

Remove From Playlist

Move To Trash
```

**Reply by noscript** [reply]
Tag Editor support requires taglib,  your build is probably missing taglib plugin. You need to configure without `--no-taglib` option.

**Reply by vitaly-zdanevich** [reply]
Removed `--no-taglib`, added dependency `media-libs/taglib`, rebuilt - still no Tag Editor in the track context menu:
![image](https://user-images.githubusercontent.com/3514015/179411222-cdcef41e-cc61-4fdb-8d90-fb951804cf69.png)

**Reply by vitaly-zdanevich** [reply]
Also tried to rebuild with `media-plugins/gst-plugins-taglib`.

**Reply by noscript** [reply]
What does nulloy print at start up? It should look something like this:
```
nulloy
...
found container "TagLib" :
* TagReader
...
registering plugin: "NTagReaderTaglib"
```

**Reply by vitaly-zdanevich** [reply]
```
$ nulloy 
found container "GStreamer" :
* PlaybackEngine
* TagReader
* WaveformBuilder
registering plugin: "NPlaybackEngineGStreamer"
registering plugin: "NWaveformBuilderGstreamer"
registering plugin: "NTagReaderGstreamer"
detected window manager: "i3"
QtScript: TypeError: Function.prototype.connect: target is not a function
```

**Reply by vitaly-zdanevich** [reply]
Another question - on your screenshot no input for unsyncedlyric?

**Reply by vitaly-zdanevich** [reply]
Rebuild again without `--no-taglib`, added dependency `media-libs/taglib`, on start from terminal:

```
found container "GStreamer" :
* PlaybackEngine
* TagReader
* WaveformBuilder
found container "TagLib" :
* TagReader
* CoverReader
registering plugin: "NPlaybackEngineGStreamer"
registering plugin: "NWaveformBuilderGstreamer"
registering plugin: "NTagReaderTaglib"
registering plugin: "NCoverReaderTaglib"
```

YES now I see tag editor!
![image](https://user-images.githubusercontent.com/3514015/182438798-a7853c14-641e-4c44-8d8a-d21386ca853e.png)

**Reply by noscript** [reply]
> Another question - on your screenshot no input for unsyncedlyric?

Not yet. Please submit a feature request.

**Reply by vitaly-zdanevich** [reply]
Submit https://github.com/nulloy/nulloy/issues/204

**Reply by vitaly-zdanevich** [reply]
I found that I have tag editor only when start from root!

**Reply by vitaly-zdanevich** [reply]
Any idea why?

**Reply by noscript** [reply]
Could it be that you run two different Nulloy builds.

**Reply by vitaly-zdanevich** [reply]
Checked by `which`: both users point to `/usr/bin/nulloy`

**Reply by noscript** [reply]
Maybe the regular user doesn't have read access to plugins?

**Reply by vitaly-zdanevich** [reply]
Hm, what can I check?

**Reply by vitaly-zdanevich** [reply]
Yes I have tag editor in root, but no sound :(
